Essentials/src/messages.properties

```diff
@@ -583,6 +583,7 @@ powerToolRemove=\u00a76Command \u00a7c{0}\u00a76 removed from \u00a7c{1}\u00a76.
 powerToolRemoveAll=\u00a76All commands removed from \u00a7c{0}\u00a76.
 powerToolsDisabled=\u00a76All of your power tools have been disabled.
 powerToolsEnabled=\u00a76All of your power tools have been enabled.
+priceChanged=\u00a76The price of \u00a7c{0}x {1}\u00a76 has changed from \u00a7e{2}\u00a76 to \u00a7e{3}\u00a76.
```

Comment:
should probably not be displayed as an error, since it isn't one. also, normally c is used for variables, not e.

Reply:
changed